tests/php/test_class.jetpack.php

```diff
@@ -476,6 +490,25 @@ function test_is_dev_version_false_with_number_dot_number_dot_number() {
 		$this->assertFalse( Jetpack::is_development_version() );
 	}
 
+	function test_get_sync_idc_option_sanitizes_out_www_and_protocol() {
```

Comment:
I switched to `get_home_url()` and `get_site_url()` so, we should always get URLs back that have a protocol.